continuous-pipe.yml

```diff
@@ -7,34 +7,57 @@ variables:
   - name: SYMFONY_ENV
     value: prod
 
+pipelines:
+  - name: Production
+    condition: 'code_reference.branch in ["master"]'
+    tasks: [ images, deployment ]
+  - name: Feature
+    condition: 'code_reference.branch matches "#^feature/#"'
+    tasks: [ images, deployment ]
+  - name: Remote
+    condition: 'code_reference.branch matches "#^cpdev/#"'
+    tasks: [ images, deployment ]
+    variables:
+      - name: SYMFONY_ENV
+        value: dev
+
 tasks:
   images:
     build:
       services:
         web:
           image: ${IMAGE_NAME}
           naming_strategy: sha1
+
   deployment:
     deploy:
       services:
         web:
+          endpoints:
+            - name: web
+              cloud_flare_zone:
+                zone_identifier: ${CLOUD_FLARE_ZONE}
+                authentication:
+                    email: ${CLOUD_FLARE_EMAIL}
+                    api_key: ${CLOUD_FLARE_API_KEY}
+                proxied: true
+                record_suffix: '-sfdemo.continuouspipe.net'
```

Comment:
I think we do, as there is no wildcard DNS pointing at the cluster it's deployed to.